features/events/show.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Detailed event page
+
+  Scenario: Tournament
+    Given a tournament with gamemode ko
+    When the tournament's page is visited
+    Then the page header should show 'Tournament'
+    And the page should show 'K.-o.-System'
+    And the page should show 'Teamteilnahme'
+
+  Scenario: League
+    Given a league with gamemode round_robin
+    When the league's page is visited
+    Then the page header should show 'League'
```

Comment:
Why is "League" english but "Jeder gegen Jeden" and "Teamteilnahme" german? Please decide for one (English would fit the rest of the sentence)